src/dmd/dmangle.d

```diff
@@ -543,25 +543,23 @@ public:
 
     static const(char)* externallyMangledIdentifier(Declaration d)
     {
-        if (!d.parent || d.parent.isModule() || d.linkage == LINKcpp) // if at global scope
+        if (!d.parent || d.parent.isModule() || d.linkage == LINK.cpp) // if at global scope
         {
-            switch (d.linkage)
+            final switch (d.linkage)
             {
-                case LINKd:
+                case LINK.d:
                     break;
-                case LINKc:
-                case LINKwindows:
-                case LINKpascal:
-                case LINKobjc:
+                case LINK.c:
+                case LINK.windows:
+                case LINK.pascal:
+                case LINK.objc:
                     return d.ident.toChars();
-                case LINKcpp:
+                case LINK.cpp:
                     return Target.toCppMangle(d);
-                case LINKdefault:
+                case LINK.def:
+                case LINK.system:
```

Comment:
I wasn't sure what the behavior of `LINK.system` should be here. FWIW `LINK.system` feels like a hack.